packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -539,6 +539,40 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
     };
 }
 
+export function fetchAllMyTeamsChannelsAndChannelMembers(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        let channels;
+        let channelsMembers;
+        const state = getState();
+        const {currentUserId} = state.entities.users;
+        try {
+            const channelRequest = Client4.getAllTeamsChannels();
+            const channelsMembersRequest = Client4.getAllChannelsMembers(currentUserId);
+            channels = await channelRequest;
+            channelsMembers = await channelsMembersRequest;
```

Comment:
What is the purpose of storing the Promises in variables here?
Wouldn't it work just as well when written like this?

```suggestion
            channels = await Client4.getAllTeamsChannels();
            channelsMembers = await Client4.getAllChannelsMembers(currentUserId);
```